package.json

```diff
@@ -17,16 +17,18 @@
   },
   "preferGlobal": true,
   "scripts": {
-    "build": "npm run tslint && gulp build:raw",
-    "tslint": "tslint --project .",
-    "test": "npm run tslint && mocha --require ts-node/register ./test/**/*-test.[tj]s",
+    "build": "npm run lint && gulp build:raw",
+    "lint": "eslint \"src/**/*.ts\" \"test/**/*.ts\"",
```

Comment:
add scripts and bin to that (if possible of course)

Reply:
Added.